CONFIGURATION.md

```diff
@@ -121,4 +121,4 @@ Environment Variable | Config File Key | Description | Default
 N/A | `:log_args` | Enable/disable the collection of URL query parameters, set to boolean false to disable. | true
 N/A | `:log_traceId` | Configure the insertion of trace context into application logs, setting `:traced` would include the available context fields such as trace_id, span_id into log messages. | `:never`
 N/A | `:tracing_mode` | Enable/disable the tracing mode for this service, setting `:disabled` would suppress all trace spans and metrics. | `:enabled`
-N/A | `:transaction_settings` | Configure tracing mode per transaction, aka transaction filtering. | None
+N/A | `:transaction_settings` | Configure tracing mode per transaction, aka transaction filtering. | None
```

Comment:
Please add the config file key for trigger tracing mode.

Reply:
Added.